ppl/src/main/antlr/OpenSearchPPLLexer.g4

```diff
@@ -54,6 +55,10 @@ STR:                                'STR';
 IP:                                 'IP';
 NUM:                                'NUM';
 
+// TRENDLINE KEYWORDS
+SMA:                                'SMA';
+WMA:                                'WMA';
```

Comment:
Maybe don't parse WMA if it is not supported/implemented yet

Reply:
Done.